setup.cfg

```diff
@@ -6,6 +6,7 @@ max-line-length = 100
 ignore=E203, W503
 per-file-ignores =
     *__init__.py:F401
+    theseus/extlib/tests/test_baspacho_simple.py:E261,E241,E121,E131,E201,E202
```

Comment:
Can you extend the comment above (lines 3-5) with the errors that these codes are referring to?

Reply:
just the fact that the simple test has matrices in the python code that I formatted in order to make the sparse pattern visible, and therefore and I don't want to be reformatted (or getting error for the format being edited manually). Will comment.

Reply:
added comment to explain.